search-books/src/main/java/nix/education/util/SymbolsSolrQuery.java

```diff
@@ -0,0 +1,7 @@
+package nix.education.util;
+
+public class SymbolsSolrQuery {
```

Comment:
I'd combine all the constant classes to a single interface SearchConstants

Reply:
I combined whole constants in SearchConstants interface